### PR TITLE
Fix use after free errors in TFDeabstraction and TFPartition.

### PR DESCRIPTION
**Pull request by bgogul**
This PR fixes two use-after-free errors found by asan.

### CODE REVIEW

**Comment by bgogul** [comment]
@swift-ci please test tensorflow linux

**Comment by bgogul** [comment]
@swift-ci please test tensorflow macos

**Review by mhong — APPROVED** [review]
Nice! Are we getting close to turning on asan in CI soonish?

File: lib/SILOptimizer/Mandatory/TFDeabstraction.cpp [hunk 1]
```diff
@@ -1860,6 +1860,7 @@ void TFDeabstraction::checkAttributesAndFormGraphOps() {
           opInfo->opName == "tfc.configureCPU")
         continue;
       formGraphOp(*opInfo, constants, deviceInfo);
+      continue;
```
[the remark this answers is not in the document]
Comment (lattner) [reply lib/SILOptimizer/Mandatory/TFDeabstraction.cpp]: I'd also add a comment right above this continue, so it doesn't look pointless.

**Review by lattner — APPROVED** [review]

**Review by bgogul — COMMENTED** [review]
> Nice! Are we getting close to turning on asan in CI soonish?
build-script already supports asan builds. This just needs to be enabled in CI as an appropriate configuration.

**Pull request merged**